pyanaconda/payload/__init__.py

```diff
@@ -113,24 +121,27 @@ def __init__(self, group, adding=True, required=False):
         self.adding = adding
         self.required = required
 
+
 class NoSuchPackage(PayloadError):
     def __init__(self, package, required=False):
         PayloadError.__init__(self, package)
         self.package = package
         self.required = required
 
+
 class DependencyError(PayloadError):
     pass
 
+
 # installation
 class PayloadInstallError(PayloadError):
     pass
 
+
 class Payload(object):
-    """ Payload is an abstract class for OS install delivery methods. """
+    """Payload is an abstract class for OS install delivery methods."""
     def __init__(self, data):
-        """ data is a kickstart.AnacondaKSHandler class
-        """
+        """Data is a kickstart.AnacondaKSHandler class."""
```

Comment:
Good point, I've just did the formatting but I didn't read the comments. Thanks I'll fix it.